cap-ci/pipeline.template

```diff
@@ -1139,6 +1147,54 @@ jobs:
             {{- print $import_aks }}
             {{- end }}
             {{- print $stratos }}
+{{ end }}
+
+{{- if index $config.jobs "clean-kubecf" }}
+- name: clean-kubecf-{{ $scheduler }}-{{ $backend }}-{{ $option }}
+  serial_groups: [{{ $scheduler }}-{{ $backend }}-{{ $option }}]
+  public: false
+  plan:
+  - get: catapult
+  - get: {{ $backend }}-pool.kube-hosts
+    passed:
+    - stratos-{{ $scheduler }}-{{ $backend }}-{{ $option }}
+    trigger: true
+  - task: clean-kubecf-{{ $scheduler }}
+    privileged: true
+    timeout: 5h30m
+    input_mapping:
+      pool.kube-hosts: {{ $backend }}-pool.kube-hosts
+    config:
+      platform: linux
+      image_resource:
+        type: registry-image
+        source:
+          repository: splatform/catapult
+      inputs:
+      - name: catapult
+      - name: pool.kube-hosts
+      params:
+        QUIET_OUTPUT: true
+        DOWNLOAD_CATAPULT_DEPS: false
```

Comment:
you need to pass eirini param for eirini deployments. Double check if you are missing any other params.

Reply:
This should be handled separately not part of this PR.

Reply:
what do you mean? Your cleanup will not work if the deployment is using eirini instead of diego. You will have to look at the cleanup script to check what parameters that script require to do the clenup.

Reply:
gotcha! I thought you were referring to Eirini deployments being configured correctly. I have spawned a test job to see if it happens correctly.

Reply:
reference: https://github.com/SUSE/cap/blob/master/cap-ci/pipeline.template#L312-L316

Reply:
> what do you mean? Your cleanup will not work if the deployment is using eirini instead of diego. You will have to look at the cleanup script to check what parameters that script require to do the clenup.

Opened https://github.com/SUSE/catapult/pull/214 for this.